code/datums/mood_events/drug_events.dm

```diff
@@ -78,3 +78,12 @@
 	description = "<span class='nicegreen'>Eh ah AAAAH! HA HA HA HA HAA! Uuuh.</span>\n"
 	mood_change = 6
 	timeout = 3 MINUTES
+
+/datum/mood_event/ldx_pain

```

Comment:
these moodlets aren't being used for anything, why are they here?

Reply:
Remove these, as LDX is no longer part of this PR.